contracts/truefi/LoanToken.sol

```diff
@@ -296,6 +321,15 @@ contract LoanToken is ILoanToken, ERC20 {
         emit Closed(status, _balance());
     }
 
+    /**
+     * @dev Liquidate the loan if it has defaulted
+     */
+    function liquidate() external override onlyDefaulted onlyLiquidator {
+        status = Status.Liquidated;
```

Comment:
Should liquidate also do reclaim?

Reply:
So i designed it the way, that if borrower payed some of his debt back, then it would be claimable with loan tokens. The difference will be taken from stkTru (or 10% bound). This way when stkTru balance is low, pool participants will suffer smaller losses.